compiler/rustc_parse/locales/en-US.ftl

```diff
@@ -220,7 +220,7 @@ parse_match_arm_body_without_braces = `match` arm body without braces
             [one] statement
            *[other] statements
         } with a body
-    .suggestion_use_comma_not_semicolon = use a comma to end a `match` arm expression
+    .suggestion_use_comma_not_semicolon = replace `;` with `,` to end this `match` arm expression
```

Comment:
great work, just taking a look and I guess this should be 
```rust
replace `;` with `,` to end a `match` arm expression
```
just to generalize it and since match expression(s) will always end with a comma `,` 🙄